docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,16 @@
 
 ## May 2024
 
+**23 May 2024**
+
+Frontend version: 1.0.0-20240520.1421 | Backend version: 1.105.3-20240521.1554
+
+| Type  | Change| Description |
+|-------|-------|-------------|
+| **Change** | Enable TechPass admin to update user email/username on identities page | Tenants can now expect to receive changes in user's email/upn via our ```user-updated webhook``` event. |
```

Comment:
for 'Change' column, no need to mention what is done by our own techpass admin, can just put 'Webhook notification for changes in user's email/upn'
Also , instead of ```user-updated webhook``` event, make it ```user-updated``` webhook event